packages/wallet/src/redux/protocols/defunding/actions.ts

```diff
@@ -16,12 +17,14 @@ import { VirtualDefundingAction, isVirtualDefundingAction } from '../virtual-def
 // -------
 
 // TODO: Replace once ledger defunding actions are defined
```

Comment:
Have the ledger defunding actions now been defined?

Reply:
yup, removed